source/2_platform/rex_directx/src/directx_feature_shader_model.cpp

```diff
@@ -0,0 +1,76 @@
+#include "rex_directx/directx_feature_shader_model.h"
+
+#include "rex_directx/directx_util.h"
+
+namespace rex
+{
+  //-------------------------------------------------------------------------
+  rsl::string shader_model_name(s32 shaderModel)
```

Comment:
can use stack string for this.
can even use enum reflection to avoid this function to begin with